plugin/plugin.cpp

```diff
@@ -50,6 +69,7 @@ void ConnmanPlugin::registerTypes(const char *uri)
         qWarning() << "MeeGo.Connman QML module name is deprecated and subject for removal. Please adapt code to \"import Connman\".";
     }
 
+    qmlRegisterSingletonType<ConnmanApi>(uri, 0, 2, "Connman", singleton_api_factory<ConnmanApi>);
```

Comment:
Misleading how? The upstream project might have its own styling of the name, but the qml import here is "Connman".

Reply:
Because it does not have anything to do with `ConnMan` itself. It is for path to using `ConnMan`.  It is a part of the library, when we change these we should rename those in a way that they clearly state where these belong. It is not a part of `ConnMan`.

Reply:
I'd consider that to be more question about the import naming than the singleton here. In retrospect could have been better if it was having some "Q" or "Qt" there, but then again connman being a lower level C project it shouldn't be that surprising that qml import is coming somewhere else.